PixelHunter1995/Inputs/InputReader.cs

```diff
@@ -15,7 +15,8 @@ class InputReader
 
         public int MouseX { get; private set; }
         public int MouseY { get; private set; }
-        public Point Position { get => new Point(this.MouseX, this.MouseY); }
+        public int MouseSceneX { get; private set; }
```

Comment:
I would prefer World over Scene. Both because Screen and Scene read so similarly, and because I am more used to that naming. Though that is purely personal preference, and as we call the other as MouseX/Y (instead of the verboose MouseWorldX/Y) there is little-to-no risk of misreading it anyway.